src/electionguard_gui/components/view_decryption_component.py

```diff
@@ -1,31 +1,100 @@
+import traceback
 from typing import Any
 import eel
-from electionguard_gui.eel_utils import eel_success
+from pymongo.database import Database
+from electionguard_gui.eel_utils import eel_fail, eel_success
 from electionguard_gui.components.component_base import ComponentBase
-from electionguard_gui.services import ElectionService, DecryptionService
+from electionguard_gui.models.decryption_dto import DecryptionDto
+from electionguard_gui.services import (
+    ElectionService,
+    DecryptionService,
+    DbWatcherService,
+)
+from electionguard_gui.services.decryption_stages import (
+    DecryptionS1JoinService,
+    DecryptionS2AnnounceService,
+)
 
 
 class ViewDecryptionComponent(ComponentBase):
```

Comment:
In a subsequent PR I'll probably split this out into one that's specific to admin and one that's specific to guardian